### PR TITLE
Add hash_meta to MerkleRoot.ToSigJSON

### PR DESCRIPTION
**Pull request by zapu**
Related to: https://github.com/keybase/keybase/pull/1329

I tried it with a few things that appended sigchains and it seems to work. Let me know if something else needs to be done there, hope I got the ticket right.

### CODE REVIEW

File: go/libkb/merkle_client.go [hunk 1]
```diff
@@ -1175,6 +1175,7 @@ func (mr *MerkleRoot) ToSigJSON() (ret *jsonw.Wrapper) {
 	ret.SetKey("seqno", jsonw.NewInt(int(*mr.Seqno())))
 	ret.SetKey("ctime", jsonw.NewInt64(int64(mr.Ctime())))
 	ret.SetKey("hash", jsonw.NewString(mr.RootHash().String()))
+	ret.SetKey("hash_meta", jsonw.NewString(mr.ShortHash().String()))
```
Comment (maxtaco) [inline go/libkb/merkle_client.go]: i'd also delete the old one

Comment (maxtaco) [inline go/libkb/merkle_client.go]: Actually, it's OK to leave both....

Comment (zapu) [inline go/libkb/merkle_client.go]: That's what I thought, for the transition period. If both are provided, both are checked by the server.

**Comment by maxtaco** [comment]
👍  Once CI passes (though is it depending on fixing the server?)

**Comment by zapu** [comment]
Looks like it won't pass CI until server fixes go in, because after moving "merkle_root" to "body" it will actually hit the `_check_merkle` in `web/sig.iced` but there is still a bug in this function (fixed by the keybase/keybase PR).

**Comment by maxtaco** [comment]
OK great, land the server first and then rerun this PR

**Pull request merged**